playground/backend/cmd/server/controller.go

```diff
@@ -79,6 +79,9 @@ func (controller *playgroundController) RunCode(ctx context.Context, info *pb.Ru
 	if err = utils.SetToCache(ctx, controller.cacheService, pipelineId, cache.RunOutputIndex, 0); err != nil {
 		return nil, errors.InternalError("Run code()", "Error during set value to cache: "+err.Error())
 	}
+	if err = utils.SetToCache(ctx, controller.cacheService, pipelineId, cache.LogsIndex, 0); err != nil {
+		return nil, errors.InternalError("Run code()", "Error during set value to cache: "+err.Error())
```

Comment:
what is the reason to use string concatenation instead of format?

Reply:
`errors.InternalError` required 2 string parameters.
So I can rewrite it to:
```
errors.InternalError("Run code()", fmt.Sprintf("Error during set value to cache: %s", err.Error()))
```

Reply:
Yeah, that's will be clearer. Just one question, internal error is our implementation?

Reply:
Yep

Reply:
changed.